content/techniques/hot-reload.md

```diff
@@ -2,17 +2,98 @@
 
 The highest impact on your application's bootstrapping process has a **TypeScript compilation**. But the question is, do we have to recompile a whole project each time when change occurs? Not at all. That's why [webpack](https://github.com/webpack/webpack) HMR (Hot-Module Replacement) significantly decreases an amount of time necessary to instantiate your application.
 
+> warning **Warning** Note that `webpack` won't automatically copy your assets (e.g. `graphql` files) to the `dist` folder. Similary, `webpack` is not compatible with glob static paths (e.g. `entities` property in `TypeOrmModule`).
+
+### With CLI
+
+If you are using [Nest CLI](http://localhost:4200/cli/overview), the configuration process should be pretty straighforward. CLI wraps `webpack` underneath, which means that you only have to apply an additional `HotModuleReplacementPlugin`.
```

Comment:
using the Nest CLI <-- add *the*
configuration process is pretty straightforward <-- replace *should be* with *is*
use: The CLI wraps `webpack`, which allows you to use the  `HotModuleReplacementPlugin`.